gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -631,7 +631,7 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         *,
         retry: OptionalRetry = gapic_v1.method.DEFAULT,
         timeout: Union[float, object] = gapic_v1.method.DEFAULT,
-        metadata: Sequence[Tuple[str, str]] = (),
+        {{ shared_macros.client_method_metadata_argument()|indent(8) }} = {{ shared_macros.client_method_metadata_default_value() }},
```

Comment:
I think these macros are no longer used and are adding a toil to whenever a change needs to be made to the client methods i.e. see the linked comment above.

I'd suggest to get rid of these macros.

Reply:
I filed https://github.com/googleapis/gapic-generator-python/issues/2255 for this clean up step. We still need to remove usage here: https://github.com/search?q=repo%3Agoogleapis%2Fgoogleapis%20add-iam-methods&type=code